Import-VIRole.ps1

```diff
@@ -0,0 +1,114 @@
+﻿function Import-VIRole
+{
+    <#  
+            .SYNOPSIS
+            Imports a vSphere role based on pre-defined configuration values
+            .DESCRIPTION
+            The Import-VIRole cmdlet is used to parse through a list of pre-defined permissions to create a new role. Often, this is to support a particular vendor's set of requirements for access into vSphere.
```

Comment:
It appears that .INPUT is rather uncommon, so I used .PARAMETER to provide details on each parameter's role. This also states the desired variable type (string) for each input when using Get-Help.